mppm/virtual-players/virtual-players-layout.md

```diff
@@ -0,0 +1,35 @@
+---
+id: virtual-players-layout
+title: Change the layout of a Virtual Player
+description: Learn how to change the layout of a Vrtual Player window.
+---
+
+You can display and use the following parts of the [Unity Interface](https://docs.unity3d.com/Manual/UsingTheEditor.html) in a Virtual Player:
+- [Console](https://docs.unity3d.com/Manual/Console.html)
+- [Game](https://docs.unity3d.com/Manual/GameView.html)
+- [Hierarchy](https://docs.unity3d.com/Manual/Hierarchy.html)
+- [Inspector](https://docs.unity3d.com/Manual/UsingTheInspector.html)
+- [Scene](https://docs.unity3d.com/Manual/UsingTheSceneView.html)
+- [PlayMode Tools](https://docs.unity3d.com/Packages/com.unity.netcode@1.2/manual/playmode-tool.html). This Window only appears in projects that use Netcode for Entities.
+
+:::note
+You can only use the Hierarchy, Inspector, and Scene windows when you enter Play Mode.
+:::
+
+## Display a window in a virtual player
+
+To display a window in a Virtual Player, perform the following actions: 
+
+1. [Activate](../virtual-players/virtual-players-enable) a Virtual Player.
+2. Select the **Layout** drop down.
+3. Check the box next to the window(s) you want to appear in this Virtual Player.
+4. Select **Apply**.
+
+## Hide a window in a virtual player
+
+To hide a window in a Virtual Player, perform the following actions: 
+
+1. [Activate](../virtual-players/virtual-players-enable) a Virtual Player.
```

Comment:
"a" virtual player, or is it "the" at this point, because it already exists?

Reply:
Unity creates the virtual player when you activate it - it doesn't exist until you do that so "a" is correct :)